src/sentry/api/endpoints/group_details.py

```diff
@@ -285,44 +279,6 @@ def get(self, request, group):
                 }
             )
 
-            # the current release is the 'latest seen' release within the
-            # environment even if it hasnt affected this issue
-            if environments:
-                with sentry_sdk.start_span(op="GroupDetailsEndpoint.get.current_release") as span:
-                    span.set_data("Environment Count", len(environments))
-                    span.set_data(
-                        "Raw Parameters",
-                        {
-                            "group.id": group.id,
-                            "group.project_id": group.project_id,
-                            "group.project.organization_id": group.project.organization_id,
-                            "environments": [{"id": e.id, "name": e.name} for e in environments],
-                        },
-                    )
-
-                    try:
-                        current_release = GroupRelease.objects.filter(
-                            group_id=group.id,
-                            environment__in=[env.name for env in environments],
-                            release_id=(
-                                ReleaseEnvironment.objects.filter(
-                                    release_id__in=ReleaseProject.objects.filter(
-                                        project_id=group.project_id
-                                    ).values_list("release_id", flat=True),
-                                    organization_id=group.project.organization_id,
-                                    environment_id__in=environment_ids,
-                                )
-                                .order_by("-first_seen")
-                                .values_list("release_id", flat=True)
-                            )[:1],
-                        )[0]
-                    except IndexError:
-                        current_release = None
-
-                data["currentRelease"] = serialize(
```

Comment:
Issue details is a documented API endpoint. You might end up breaking external integrations with this removal. cc @getsentry/ecosystem

Reply:
We did investigate that at the time and determined that `currentRelease` is internal-only, or at least that seems to be the intent based on the code history and all available documentation. If it were being consumed externally, I would expect the rather obscure meaning of null/non-null values to be explained somewhere.

But, 👍 to getting approval from @getsentry/ecosystem.

Reply:
@RyanSkonnord Nah, Ecosystem does not even use the GET method on this endpoint. We do stuff for updating the issue/group to be resolved in the current release but that's a PUT request.